examples/src/main/scala/examples/commons/SimpleBoxTransaction.scala

```diff
@@ -122,7 +123,7 @@ object SimpleBoxTransaction extends ScorexEncoding {
     val amount = to.map(_._2.toLong).sum
 
     val from: IndexedSeq[(PrivateKey25519, Nonce, Value)] = w.boxes()
-      .filter(b => !boxesIdsToExclude.exists(_ sameElements b.box.id)).sortBy(_.createdAt).takeWhile { b =>
+      .filter(b => !boxesIdsToExclude.contains(b.box.id)).sortBy(_.createdAt).takeWhile { b =>
```

Comment:
if we need to be faster here, we could use Sets.

Reply:
It's ok for examples